src/it/java/net/spy/memcached/AutoDiscoveryTest.java

```diff
@@ -28,15 +29,15 @@ public class AutoDiscoveryTest {
   
   @Before
   public void setUp() throws Exception {
-    cf = new DefaultConnectionFactory() {
+    cf = new ClientTestConnectionFactory() {
       @Override
```

Comment:
You can remove this method. The implementation is the same as the base class.

Reply:
As discussed, `AutoDiscoveryTest` doesn't extend from `ClientBaseCase`

Reply:
Remove duplicated methods if the implementation is the same as in `ClientTestConnectionFactory`